exchange/exchange.go

```diff
@@ -186,6 +186,7 @@ func (e *exchange) getAllBids(ctx context.Context, cleanRequests map[openrtb_ext
 	// Wait for the bidders to do their thing
 	for i := 0; i < len(cleanRequests); i++ {
 		brw := <-chBids
+		brw.validateBids()
```

Comment:
If the `bidResponseWrapper` has `validateBids()` called on it inside of the goroutine before being sent back through the channel, we shouldn't have to call it a second time here.

Reply:
Oops, yes, I had moved it from outside to inside, and forgot to get rid of this line. Thanks.